core/src/FS.purs

```diff
@@ -15,6 +15,7 @@ module Spago.FS
   , readTextFile
   , readYamlDocFile
   , readYamlFile
+  , rmRf
```

Comment:
I wouldn't like this in application code - it looks like it's only used for tests so we should move it into the Test Prelude or just in the `Test.Spago.Build.Monorepo` module